templates/main.bicep

```diff
@@ -0,0 +1,358 @@
+targetScope = 'resourceGroup'
+
+// General parameters
+param location string = resourceGroup().location
+param tenantId string = subscription().tenantId
+param odhsiWebApiName string = 'ohdsi-webapi'
+param dockerRegistryServer string = 'https://index.docker.io/v1'
+param dockerImageName string = 'ohdsi/webapi'
+param dockerImageTag string = '2.12.1'
+param keyvaultName string ='keyvault-${uniqueString(resourceGroup().id)}'
+
+// Postgres parameters
+param serverEdition string = 'GeneralPurpose'
+param skuSizeGB int = 32
+param dbInstanceType string = 'Standard_D2s_v3'
+param availabilityZone string = '3'
+param version string = '14'
+param postgresDatabaseName string = 'atlas_db'
```

Comment:
We will have multiple DBs in this server and probably the param name should reflect the usecase.

Reply:
Fixed - now is atlas_webapi_db